index.html

```diff
@@ -161,123 +114,134 @@ <h2>
+            </ol>
+          </li>
+          <li>Create a new <a>PerformanceMeasure</a> object (<var>entry</var>).</li>
+          <li>Set <var>entry</var>'s <code>name</code> property to <var>measureName</var>.</li>
+          <li>Set <var>entry</var>'s <code>entryType</code> property to <code>DOMString "measure"</code>.</li>
+          <li>Set <var>entry</var>'s <code>startTime</code> property to <var>start time</var>.</li>
+          <li>Set <var>entry</var>'s <code>duration</code> property to the duration from <var>start time</var> to <var>end time</var>. The resulting duration value MAY be negative.</li>
+          <li><a data-cite="!PERFORMANCE-TIMELINE-2#dfn-queue-a-performanceentry">Queue</a> <var>entry</var>.</li>
+          <li id="stored_measure">Add <var>entry</var> to the <a data-cite="!PERFORMANCE-TIMELINE-2#dfn-performance-entry-buffer">performance entry buffer</a>.</li>
+          <li>Return <strong>undefined</strong>.</li>
+        </ol>
+      </section>
+      <section>
+        <h2><dfn>clearMeasures()</dfn> method</h2>
+        <p>Removes stored timestamp with the associated name. It MUST run these steps:</p>
+        <ol>
+          <li>If <var>measureName</var> is omitted, remove all <a>PerformanceMeasure</a> objects in the <a data-cite="!PERFORMANCE-TIMELINE-2#dfn-performance-entry-buffer">performance entry buffer</a>.</li>
+          <li>Otherwise remove all <a>PerformanceMeasure</a> objects listed in the <a data-cite="!PERFORMANCE-TIMELINE-2#dfn-performance-entry-buffer">performance entry buffer</a> whose <code>name</code> matches <var>measureName</var>.</li>
+          <li>Return <strong>undefined</strong>.</li>
+        </ol>
+      </section>
+    </section>
+    <section id="performancemark" data-dfn-for="PerformanceMark" data-link-for="PerformanceMark">
+      <h2>The <dfn>PerformanceMark</dfn> Interface</h2>
+      <p>The <a>PerformanceMark</a> interface also exposes marks created via the <a>performance.mark</a> method to the <a data-cite="!PERFORMANCE-TIMELINE-2#dfn-performance-timeline">Performance Timeline</a>.</p>
+      <pre class="idl">
+        [Exposed=(Window,Worker)]
+        interface PerformanceMark : PerformanceEntry {
+        };
+      </pre>
+      <p>The <a>PerformanceMark</a> interface extends the following attributes of the <a data-cite="!PERFORMANCE-TIMELINE-2#performanceentry">PerformanceEntry</a>
+      interface:</p>
+      <p>The <code>name</code> attribute will return the mark's name.</p>
+      <p>The <code>entryType</code> attribute will return the <a data-cite="!WEBIDL#idl-DOMString"><code>DOMString</code></a> <code>"mark"</code>.</p>

```

Comment:
Are these redefinitions?

Reply:
I'm not quite sure what you mean by that, but regardless, I didn't make any content changes here. Maybe @igrigorik can comment?

Reply:
Sorry, let me be more clear. This section is about `interface PerformanceMark`, then it talks about `entryType` attribute. Where is `entryType` defined? Why is it being discussed here?

Reply:
That is, `PerformanceEntry` defines the `entryType` attribute (which is in a different spec), thus this spec should not (re)define the `entryType` attribute. That's not great, imo.

Reply:
@marcoscaceres PerformanceMark inherits entryType from PerformanceEntry, and we're defining what PerformanceMark should return for entryType; this use is consistent with how we intended this to be defined and the pattern we've followed across all of our specs.

I don't see any issues here, but open to suggestions if there's a better way to do this. That said, given that this is a standard pattern we've been using, we don't need to block this update on this. If there's a better way to do this, let's take that as a meta issue.